Detections/AzureActivity/Creation_of_Expensive_Computes.yaml

```diff
@@ -0,0 +1,36 @@
+id: 9736e5f1-7b6e-4bfb-a708-e53ff1d182c3
+name: Creation of expensive computes in Azure
```

Comment:
Just go with the full name in the file name or change the name here to `Creation of expensive computes` to match the file name.